index.html

```diff
@@ -31,7 +31,19 @@
             </li>
           </ul>
         </div>
-        <img src="./images/hamburger.svg" alt="menu-icon" class="menu" />
+        <button type="button" id="menu-btn">
+          <img src="./images/hamburger.svg" alt="menu-icon" class="menu" />
+        </button>
+        <div class="menu-container">
+          <div class="image-container">
+            <img src="./images/Icon.svg" alt="close-menu-icon" />
+          </div>
+          <div class="nav-links">
+            <a href="#works-section">Portoflio</a>
+            <a href="#about-section">About</a>
+            <a href="#contact-me-section">Contact</a>
+          </div>
+        </div>
```

Comment:
- [ ] Please ensure that the mobile menu is closed when the user first opens the page. Currently, the mobile menu is opened by itself as soon as the page reloads.